soc/riscv/riscv-privilege/gd32vf103/Kconfig.defconfig.gd32vf103

```diff
@@ -10,10 +10,7 @@ config KERNEL_ENTRY
 	default "__nuclei_start"
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
-	default $(dt_node_int_prop_int,/cpus/cpu@0,clock-frequency) if RISCV_MACHINE_TIMER
-
-config RISCV_MACHINE_TIMER_SYSTEM_CLOCK_DIVIDER
-	default $(dt_node_int_prop_int,/soc/timer@d1000000,clk-divider) if RISCV_MACHINE_TIMER
+	default 27000000
```

Comment:
This may require a help message to explain from where the 27MHz came.

Reply:
None of the Kconfig.defconfig.* files have a help section.
It shouldn't be made here either.
I put a comment instead.